### PR TITLE
[React] New Card component

### PR DESCRIPTION
**Pull request by zaliqarosli**
## Brief summary of changes

very simple. essentially just uses the already existing Panel component but adds onclick functionality

### CODE REVIEW

**Comment by driusan** [comment]
I'm a little confused by the 'onClick'.. isn't there already an onClick? It seems to just propagate it down.

**Comment by zaliqarosli** [comment]
@driusan the Panel component has an onClick for expanding/collapsing the panel. the 
Card component's onClick activates when you click on the body of the card e.g.  If you click on the 'Imaging' Card, it can take you to the Imaging browser

File: jsx/Card.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  handleClick(e) {
+    this.props.onClick(e);
+  }
+
+  render() {
+    const cursorStyle = this.props.onClick ? {
+      cursor: 'pointer',
+    } : null;
+    return (
+      <div style={{marginLeft: '5px', marginRight: '5px'}}>
```
Comment (maltheism) [inline jsx/Card.js]: Hey @zaliqarosli, I believe the best practice is to have the style object for marginLeft & marginRight defined outside the div as a constant.

**Comment by HenriRabalais** [comment]
I'm a little confused by this as well. If the only change is adding an onClick to the Panel body, couldn't this be a new prop that can be passed to the Panel component?

**Comment by zaliqarosli** [comment]
@HenriRabalais the Card component is essentially a wrapper for the Panel class with css on top of the onClick change so that they can be rendered next to each other nicely. I could have just used the Panel class and added a margin outside each one but I thought this was cleaner. I also thought that as we move forward with our UI/UX road map, there might be a need for a separate Card component to the Panel one. In my opinion, the two components provide different services so deserve their own class. It just so happens that the Panel class already had what I was looking for UI wise so I reused it! :)

**Comment by driusan** [comment]
Failing Travis

File: jsx/Card.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            {this.props.children}
+          </div>
+        </Panel>
+      </div>
+    );
+  }
+}
+
+Card.propTypes = {
+  id: PropTypes.string.isRequired,
```
Comment (driusan) [inline jsx/Card.js]: I'm trying to use this component and I don't see why `id` is required. It doesn't seem to be declared are required by the Panel that it's being passed to, so why add this requirement for cards?

Comment (zaliqarosli) [reply jsx/Card.js]: Id is required for collapsing the intended panel. If all the cards on one page have panel component with id 'default-panel', trying to collapse any one card will always collapse the first card

Comment (driusan) [reply jsx/Card.js]: I don't understand why id is required for collapsing the panel, the collapsed state is in the state of the panel. The fact that it's required and if it's not passed or distinct the wrong panel is collapsed is why I find it confusing that you need to pass an id when trying to use this card.

Comment (zaliqarosli) [reply jsx/Card.js]: The code for collapsing the panel is in the panel component and it targets an element with the given id. If there are multiple elements with the same Id i.e. the default one, then the collapsing function can't select the correct element

Comment (driusan) [reply jsx/Card.js]: Okay, I guess that's a separate issue that should be fixed outside of this PR.

**Pull request merged**